[2022-02-03 18:19:25,565] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: fhv_gcs_dag.delete_the_local_file scheduled__2019-11-02T00:00:00+00:00 [queued]>
[2022-02-03 18:19:25,634] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: fhv_gcs_dag.delete_the_local_file scheduled__2019-11-02T00:00:00+00:00 [queued]>
[2022-02-03 18:19:25,634] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 18:19:25,634] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-03 18:19:25,634] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 18:19:25,688] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): delete_the_local_file> on 2019-11-02 00:00:00+00:00
[2022-02-03 18:19:25,700] {standard_task_runner.py:52} INFO - Started process 1285 to run task
[2022-02-03 18:19:25,712] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'fhv_gcs_dag', 'delete_the_local_file', 'scheduled__2019-11-02T00:00:00+00:00', '--job-id', '190', '--raw', '--subdir', 'DAGS_FOLDER/fhv_gcs_dag.py', '--cfg-path', '/tmp/tmps01g4y05', '--error-file', '/tmp/tmptfpldzfg']
[2022-02-03 18:19:25,713] {standard_task_runner.py:77} INFO - Job 190: Subtask delete_the_local_file
[2022-02-03 18:19:25,963] {logging_mixin.py:109} INFO - Running <TaskInstance: fhv_gcs_dag.delete_the_local_file scheduled__2019-11-02T00:00:00+00:00 [running]> on host bd2f56b76e32
[2022-02-03 18:19:26,069] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-03 18:19:26,148] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=fhv_gcs_dag
AIRFLOW_CTX_TASK_ID=delete_the_local_file
AIRFLOW_CTX_EXECUTION_DATE=2019-11-02T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-11-02T00:00:00+00:00
[2022-02-03 18:19:26,150] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-03 18:19:26,150] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'rm /opt/***/output_2019-11.csv | rm /opt/***/output_2019-11.parquet']
[2022-02-03 18:19:26,168] {subprocess.py:85} INFO - Output:
[2022-02-03 18:19:26,318] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-03 18:19:26,366] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=fhv_gcs_dag, task_id=delete_the_local_file, execution_date=20191102T000000, start_date=20220203T181925, end_date=20220203T181926
[2022-02-03 18:19:26,490] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-03 18:19:26,621] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
<<<<<<< HEAD
[2022-02-19 23:59:56,713] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: fhv_gcs_dag.delete_the_local_file scheduled__2019-11-02T00:00:00+00:00 [queued]>
[2022-02-19 23:59:56,731] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: fhv_gcs_dag.delete_the_local_file scheduled__2019-11-02T00:00:00+00:00 [queued]>
[2022-02-19 23:59:56,731] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-19 23:59:56,731] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-19 23:59:56,731] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-19 23:59:56,779] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): delete_the_local_file> on 2019-11-02 00:00:00+00:00
[2022-02-19 23:59:56,792] {standard_task_runner.py:52} INFO - Started process 280 to run task
[2022-02-19 23:59:56,797] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'fhv_gcs_dag', 'delete_the_local_file', 'scheduled__2019-11-02T00:00:00+00:00', '--job-id', '307', '--raw', '--subdir', 'DAGS_FOLDER/fhv_gcs_dag.py', '--cfg-path', '/tmp/tmpkr63hlj_', '--error-file', '/tmp/tmpq9f171r8']
[2022-02-19 23:59:56,798] {standard_task_runner.py:77} INFO - Job 307: Subtask delete_the_local_file
[2022-02-19 23:59:56,944] {logging_mixin.py:109} INFO - Running <TaskInstance: fhv_gcs_dag.delete_the_local_file scheduled__2019-11-02T00:00:00+00:00 [running]> on host da07aa254901
[2022-02-19 23:59:57,046] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-19 23:59:57,098] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=fhv_gcs_dag
AIRFLOW_CTX_TASK_ID=delete_the_local_file
AIRFLOW_CTX_EXECUTION_DATE=2019-11-02T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-11-02T00:00:00+00:00
[2022-02-19 23:59:57,100] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-19 23:59:57,101] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'rm /opt/***/output_2019-11.csv | rm /opt/***/output_2019-11.parquet']
[2022-02-19 23:59:57,121] {subprocess.py:85} INFO - Output:
[2022-02-19 23:59:57,146] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-19 23:59:57,197] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=fhv_gcs_dag, task_id=delete_the_local_file, execution_date=20191102T000000, start_date=20220219T235956, end_date=20220219T235957
[2022-02-19 23:59:57,267] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-19 23:59:57,305] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-02-19 23:59:57,339] {dagrun.py:545} INFO - Marking run <DagRun fhv_gcs_dag @ 2019-11-02 00:00:00+00:00: scheduled__2019-11-02T00:00:00+00:00, externally triggered: False> successful
[2022-02-19 23:59:57,339] {dagrun.py:605} INFO - DagRun Finished: dag_id=fhv_gcs_dag, execution_date=2019-11-02 00:00:00+00:00, run_id=scheduled__2019-11-02T00:00:00+00:00, run_start_date=2022-02-19 23:59:13.627531+00:00, run_end_date=2022-02-19 23:59:57.339581+00:00, run_duration=43.71205, state=success, external_trigger=False, run_type=scheduled, data_interval_start=2019-11-02 00:00:00+00:00, data_interval_end=2019-12-02 00:00:00+00:00, dag_hash=955118f9070d82ffa6f8c965ceb83d88
[2022-02-23 20:44:14,825] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: fhv_gcs_dag.delete_the_local_file scheduled__2019-11-02T00:00:00+00:00 [queued]>
[2022-02-23 20:44:14,848] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: fhv_gcs_dag.delete_the_local_file scheduled__2019-11-02T00:00:00+00:00 [queued]>
[2022-02-23 20:44:14,848] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-23 20:44:14,848] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-23 20:44:14,849] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-23 20:44:14,871] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): delete_the_local_file> on 2019-11-02 00:00:00+00:00
[2022-02-23 20:44:14,878] {standard_task_runner.py:52} INFO - Started process 586 to run task
[2022-02-23 20:44:14,883] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'fhv_gcs_dag', 'delete_the_local_file', 'scheduled__2019-11-02T00:00:00+00:00', '--job-id', '47', '--raw', '--subdir', 'DAGS_FOLDER/fhv_gcs_dag.py', '--cfg-path', '/tmp/tmpa9_2tz17', '--error-file', '/tmp/tmpniy94w01']
[2022-02-23 20:44:14,884] {standard_task_runner.py:77} INFO - Job 47: Subtask delete_the_local_file
[2022-02-23 20:44:14,957] {logging_mixin.py:109} INFO - Running <TaskInstance: fhv_gcs_dag.delete_the_local_file scheduled__2019-11-02T00:00:00+00:00 [running]> on host 1c172e0325c8
[2022-02-23 20:44:15,019] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-23 20:44:15,048] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=fhv_gcs_dag
AIRFLOW_CTX_TASK_ID=delete_the_local_file
AIRFLOW_CTX_EXECUTION_DATE=2019-11-02T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-11-02T00:00:00+00:00
[2022-02-23 20:44:15,049] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-23 20:44:15,050] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'rm /opt/***/output_2019-11.csv | rm /opt/***/output_2019-11.parquet']
[2022-02-23 20:44:15,067] {subprocess.py:85} INFO - Output:
[2022-02-23 20:44:15,084] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-23 20:44:15,116] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=fhv_gcs_dag, task_id=delete_the_local_file, execution_date=20191102T000000, start_date=20220223T204414, end_date=20220223T204415
[2022-02-23 20:44:15,175] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-23 20:44:15,247] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
=======
>>>>>>> origin/master
